rpcserver.go

```diff
@@ -2938,8 +2939,12 @@ func (r *rpcServer) SubscribeChannelEvents(req *lnrpc.ChannelEventSubscription,
 			var update *lnrpc.ChannelEventUpdate
 			switch event := e.(type) {
 			case channelnotifier.OpenChannelEvent:
-				channel := createRPCOpenChannel(r, graph,
+				channel, err := createRPCOpenChannel(r, graph,
 					event.Channel, true)
+				if err != nil {
```

Comment:
There can't be a race condition here? That the fitness subsystem isn't ready yet / hasn't processed the open channel event and returns an err?

Reply:
I think it does, I suspect this is why travis is failing. 

I am going to export the channel not found error and only error in other cases.

Reply:
Ok, sounds safe. Maybe in the future, this uptime stat will be moved somewhere else anyway.